physical/map.go

```diff
@@ -51,6 +52,40 @@ func (node *Map) Materialize(ctx context.Context, matCtx *MaterializationContext
 	return execution.NewMap(matExprs, materialized, node.Keep), nil
 }
 
+func isVariableNameRecursive(expr Expression, name octosql.VariableName) bool {
```

Comment:
It’s not quite clear to me what this function does - a comment would be appreciated :)

Reply:
Fixed